src/components/Text/Text.tsx

```diff
@@ -0,0 +1,43 @@
+// import "./Text.css";
+// import "./_Text.scss";
+import errorSvg from '../../asset/icon/error.svg';
+import searchSvg from '../../asset/icon/search.svg';
+import React, { InputHTMLAttributes, FC } from 'react';
+
+
+interface ITextProps {
+  /** set customized error */
+  error?: boolean;
+}
+
+// function Text(props: any) {
+const Text: FC<ITextProps> = (props) => {
+  const { error, ...rest } = props;
```

Comment:
Need value as props, as well as an onchange function in case value is sent by parent.